tests/lms/views/decorators/test_h_api.py

```diff
@@ -330,13 +235,6 @@ def pyramid_request(self, pyramid_request):
         pyramid_request.params["context_id"] = "test_context_id"
         return pyramid_request
 
-    @pytest.fixture
-    def models(self, models):
-        models.CourseGroup.get.return_value = mock.create_autospec(
-            CourseGroup, instance=True, pubid="test_pubid"
-        )
-        return models
-
 
 @pytest.fixture
 def context():
```

Comment:
General ongoing curiosity: I think I would have tried to mock/auto-spec the context here instead of hand-creating a context. Is there a reason for doing it this way? (Honestly just curiosity; not suggesting it's wrong!)